tools/run-e2e.mjs

```diff
@@ -181,8 +188,9 @@ if (argv.k) {
 const e2eProcess = child_process.spawn('pipenv', e2eArgs, {
   stdio: ['inherit', 'pipe', 'pipe'],
   env: {
-    BROWSER_BIN: installAndGetChromePath(),
     ...process.env,
+    BROWSER_BIN: installAndGetChromePath(),
```

Comment:
I think this should stay where it is else, the provided BROWSER_BIN will be wrong in the case where you want to run custom binary.

Reply:
`installAndGetChromePath` respects the `BROWSER_BIN` env variable, so I guess we should force to use one returned by `installAndGetChromePath`